utils/src/main/kotlin/DiskCache.kt

```diff
@@ -45,7 +45,7 @@ class DiskCache(
         /**
```

Comment:
Commit message nit: "property" instead of "member variable".

Reply:
Done.